src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor.cs

```diff
@@ -7,16 +7,34 @@ namespace Bit.Client.Web.BlazorUI
 {
     public partial class BitNav
     {
+        /// <summary>
+        /// The key of the nav item selected by caller
+        /// </summary>
         [Parameter] public string? SelectedKey { get; set; }
 
+        /// <summary>
+        /// The aria label of nav container for the benefit of screen readers
+        /// </summary>
         [Parameter] public string? AriaLabel { get; set; }
 
+        /// <summary>
+        /// Indicates whether the navigation component renders on top of other content in the UI
+        /// </summary>
         [Parameter] public bool IsOnTop { get; set; }
 
+        /// <summary>
+        /// A collection of link items to display in the navigation bar
+        /// </summary>
         [Parameter] public ICollection<BitNavLinkItem> NavLinkItems { get; set; } = new List<BitNavLinkItem>();
 
+        /// <summary>
+        /// Callback invoked when a link in the navigation is clicked
+        /// </summary>
         [Parameter] public EventCallback<BitNavLinkItem> OnClick { get; set; }
 
+        /// <summary>
+        /// The template of navigation header, it can be a BitNavLinkItem
```

Comment:
what does it mean that "it can be a BitNavLinkItem"?
It seems wrong

Reply:
I'm trying to write it differently.